core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -631,6 +631,20 @@ secret_configs:
       - cluster: api.ci
         namespace: ci
         name: operator-manifests-test-credentials
+  # pull secret on build farms
+  - from:
+      ".dockerconfigjson":
+        bw_item: quay.io
+        attachment: pull-secret
+    to:
+      - cluster: build01
```

Comment:
@hongkailiu why build01,02 and not all build clusters via the group?

/hold

Reply:
This secret is used on OCP4 only. So api.ci is out.
For `vphere`: @jcpowermac used its own pull secret. I do not want to overwrite his setup.

Reply:
If he uses its own pull secret, random things will not work. What is the disadvantage of using it on api.ci?

I want all build clusters to be equal and half of them be a snowflake

Reply:
@hongkailiu I would think it should be consistant between all the clusters right?

Reply:
In other words I would expect the only difference between vsphere and build{01,02} is that vsphere just runs vsphere-based jobs

Reply:
OK. We are on the same page.

```
oc --context api.ci get ns openshift-config
Error from server (NotFound): namespaces "openshift-config" not found
```

I will create the ns on `api.ci`.

Reply:
```
oc --context app.ci get secret -n openshift-config pull-secret -o yaml > app.ci.openshift-config.pull-secret.secret.yaml

oc --kubeconfig ~/Downloads/joe2/sa.config-updater.vsphere.config get secret -n openshift-config pull-secret -o yaml > vsphere.openshift-config.pull-secret.secret.yaml
```